src/main/default/aura/Lookup/LookupController.js

```diff
@@ -10,11 +11,13 @@
         action.setCallback(this, (response) => {
             const state = response.getState();
             if (state === 'SUCCESS') {
+                helper.toggleSearchSpinner();
                 // Process server success response
                 const returnValue = response.getReturnValue();
                 component.set('v.searchResults', returnValue);
             }
             else if (state === 'ERROR') {
+                helper.toggleSearchSpinner();
```

Comment:
Pass `component` to `helper.toggleSearchSpinner();`

Reply:
@pozil whoops, sorry - as I mentioned I wasn't able to test this before submitting it, or would have caught that (good example of why you shouldn't submit un-tested code!). I meant to test the next day and update as necessary but slipped my mind. I will fix this and update shortly.